lib/components/Callout.js

```diff
@@ -13,16 +14,22 @@ const calloutIcons = {
   danger: "ri-spam-fill"
 };
 
-const Callout = ({ icon, showIcon, style = "info", children, className }) => {
+const Callout = ({ icon = null, style = "info", children, className = "" }) => {
```

Comment:
@yedhink _a className works fine with passing default params.
```suggestion
const Callout = ({ icon = null, style = "info", children, className }) => {
```

Reply:
@edwinbbu Yes it does. But the user can also not provide the `className` attribute while creating a `Callout` component. Thus it's a good practice to write it like this, so that it automatically becomes non-mandatory/non-required component. This also helps with the intellisense. If we keep it as just `className`, then we will get an error in the text editor too. But more than text editor, I feel it's a good practice. That's all.

If there are any caveats with doing this, then let me know. Other wise I think we can keep this. No harm. Only advantages.

@edwinbbu Let me know your thoughts.

Reply:
@yedhink I feel the code readability reduces when providing default params like `""` or `null`. Especially on the first look. Providing default params like `true` or `false` makes sense. Let's not worry of intellisense. It's up to the person to use any editor/plugin.

Reply:
@edwinbbu In my opinion the code readability is enhanced when we read through it. Let's say it was like this:
```
const Callout = ({ icon, style = "info", children, className }) => {
```
So if I'm someone starting out new in dealing with this component or if i'm using this component and then when in doubt, just checking the source to code too see how it's working, then wouldn't it be easier to say make `icon = null`, so that the user can easily recognise that it's not a required attribute? I mean I wouldn't even have to go further through the rest of the code. 

Since we don't have Prop types yet, I think this would be a very good coding standard.

If you are still not convinced, then let me know. I will definitely revert. :smile:

Reply:
We can have the `= ""` and `= null` default values.